pkg/thing/mocks/connector.go

```diff
@@ -29,3 +30,9 @@ func (fc *FakeConnector) SendUpdateSchema(id string, schemaList []entities.Schem
 	ret := fc.Called(id, schemaList)
 	return ret.Error(0)
 }
+
+// SendPublishData provides a mock function to send an publish data command to connector
```

Comment:
"SendPublishData provides a mock function to send **a** publish data command to connector"

Reply:
Done!